torch/autograd/profiler.py

```diff
@@ -606,9 +606,7 @@ def __init__(self, name: str):
         self.name: str = name
         # Whether or not we should run record function's end callbacks when exiting.
         self.run_callbacks_on_exit: bool = True
-        # Stores underlying RecordFunction as a tensor. TODO: move to custom
-        # class (https://github.com/pytorch/pytorch/issues/35026).
-        self.handle: torch.Tensor = torch.zeros(1)
+        self.handle: torch.classes.profiler._RecordFunctionHolder()
```

Comment:
Does this work? IIUC this just defines the type, we need to actually initialize it?

Reply:
Great catch.